server/prisma/migrations/20220928202622_chage_json_b_type/migration.sql

```diff
@@ -0,0 +1,8 @@
+/*
+  Warnings:
+
+  - Made the column `timetableData` on table `Timetable` required. This step will fail if there are existing NULL values in that column.
+
+*/
+-- AlterTable
+ALTER TABLE "Timetable" ALTER COLUMN "timetableData" SET NOT NULL;
```

Comment:
Does this migration just undo the previous one? If that is true can't we just remove both of them and keep the first one?